### PR TITLE
Add conda-tree

### PR DESCRIPTION
**Pull request by xylar**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by xylar** [comment]
@rvalieris, I've used conda-tree and found it very useful.  I know you host it on your own anaconda channel but I thought it would be useful on conda-forge.  Let me know if you would like me to include you as a maintainer.  I'm fine either way.

**Comment by rvalieris** [comment]
hello, nice to see this little tool is being useful. yes, add me as maintainer please.

**Comment by xylar** [comment]
> yes, add me as maintainer please.

Done.  Thanks!

**Comment by xylar** [comment]
@conda-forge/help-python, this is ready to review.  It is a pure python package so failing appveyor and azure/win_64 is not expected to be a problem.

File: recipes/conda-tree/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  noarch: python
+  script:    # See https://github.com/conda/conda-build/issues/3166
+    - mkdir -p "$PREFIX/bin"
+    - cp conda-tree.py "$PREFIX/bin/conda-tree"
+
+requirements:
+  host:
+    - python
+  run:
+    - networkx
```
Comment (mariusvniekerk) [inline recipes/conda-tree/meta.yaml]: I'd add in python as a run time dep as well

File: recipes/conda-tree/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+source:
+  url: https://github.com/rvalieris/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 0a2adacc7416d02b6a8a923bc23762655d0c28dd4f0498f3cd72ba3bffc4a1e7
+
+build:
+  number: 0
+  noarch: python
+  script:    # See https://github.com/conda/conda-build/issues/3166
+    - mkdir -p "$PREFIX/bin"
+    - cp conda-tree.py "$PREFIX/bin/conda-tree"
```
Comment (mariusvniekerk) [inline recipes/conda-tree/meta.yaml]: This will not work for windows.
Can you make this a python installable package and use the scripts entrypoint?

Comment (xylar) [reply recipes/conda-tree/meta.yaml]: Thanks @mariusvniekerk.  This is not my repo so I was just following @rvalieris's lead but I can now see why that wouldn't work in Windows.

@rvalieris, are you interested in making these changes.  If not, I can make a pull request to your branch if that would be okay.  Much less preferred, I could make a patch here.

File: recipes/conda-tree/meta.yaml (hunk 1 above)
Comment (xylar) [reply recipes/conda-tree/meta.yaml]: I'll happily do that. I had incorrectly thought that packages in host were automatically in run.

File: recipes/conda-tree/meta.yaml (hunk 2 above)
Comment (xylar) [reply recipes/conda-tree/meta.yaml]: @rvalieris, I'm going to go ahead and make a PR to your repo.

Comment (xylar) [reply recipes/conda-tree/meta.yaml]: See https://github.com/rvalieris/conda-tree/pull/4

Comment (rvalieris) [reply recipes/conda-tree/meta.yaml]: just merged the PR, I don't use windows, but feel free to send more PRs if needed.

Comment (chrisburr) [reply recipes/conda-tree/meta.yaml]: @rvalieris Can you tag a `v0.0.3` release as well?

Comment (rvalieris) [reply recipes/conda-tree/meta.yaml]: done !

**Comment by xylar** [comment]
@chrisburr, any idea why appveryor is suddenly the only CI running?  I'm going to make a trivial change and see if I can get the others going again.

**Comment by chrisburr** [comment]
I'm not sure but I've also has the same thing in other PRs so I think it's a temporary Azure issue.

**Comment by xylar** [comment]
Well, appveyor is useful for the moment because it was the windows build that was failing before.

**Comment by xylar** [comment]
@chrisburr, could you ping me (maybe on Gitter) when Azure is working again and I can trigger another build here? It's a pretty dead-simple package but it would still be best to have a linux and OSX check just in case.

**Comment by xylar** [comment]
Thanks again, @chrisburr!

**Pull request merged**